tests/manage/z_cluster/test_bz.py

```diff
@@ -0,0 +1,26 @@
+import logging
```

Comment:
whats the purpose of this test case

Reply:
it is only for testing . 
I want to check mg dir content after test failed

Reply:
So it will be removed, right?

Reply:
correct